sentry/src/test/java/io/sentry/SpanTest.kt

```diff
@@ -486,6 +486,14 @@ class SpanTest {
         assertEquals(1, transaction.root.measurements["test"]!!.value)
     }
 
+    @Test
+    fun `span provides local metrics aggregator instance`() {
+        val span = fixture.getSut()
+        assertNotNull(span.localMetricsAggregator)
+
+        assertSame(span.localMetricsAggregator, span.localMetricsAggregator)
```

Comment:
is it a mistake?

Reply:
Nope this is intended, it just ensures the getter returns a reference to the same `LocalMetricsAggregator`.
Let me add a comment for this 👍